src/common/process/ProcessUtils.cpp

```diff
@@ -167,4 +170,40 @@ StatusOr<std::string> ProcessUtils::runCommand(const char* command) {
     return out.str();
 }
 
+
+double ProcessUtils::getMemUsage() {
+    struct rusage r_usage;
+    getrusage(RUSAGE_SELF,&r_usage);
+    return r_usage.ru_maxrss;
```

Comment:
Please investigate what `ru_maxrss` really means.
Moreover, memory usage is not only about phsical memory. Please be more specific on the function name.

Reply:
I fixed this, I fetch the memory usage from `/proc/self/stat`